Tasks/AzureAppServiceManage/task.json

```diff
@@ -17,14 +17,15 @@
     "version": {
         "Major": 0,
         "Minor": 2,
-        "Patch": 22
+        "Patch": 23
     },
     "minimumAgentVersion": "1.102.0",
     "instanceNameFormat": "$(Action): $(WebAppName)",
     "groups": [],
     "inputs": [
         {
             "name": "ConnectedServiceName",
+            "aliases": ["azureSubscription"],
```

Comment:
Should we make aliases to be UpperCamelCase as same as Input names?

Reply:
The naming convention was discussed and settled with current form. Adding @RoopeshNair who arrived at this format.